source/simulator/initial_conditions.cc

```diff
@@ -173,6 +174,18 @@ namespace aspect
 
 
   template <int dim>
+  void Simulator<dim>::initialize_tracers_if_selected ()
+  {
+    Postprocess::Tracers<dim> *tracer_postprocessor = const_cast<Postprocess::Tracers<dim> *>
+                                                      (postprocess_manager.template find_postprocessor<Postprocess::Tracers<dim> >());
+
+    // If the tracer postprocessor has been selected
+    if (tracer_postprocessor != 0)
+      tracer_postprocessor->generate_and_initialize_particles();
```

Comment:
This function looks good. It's not overly elegant to use the `const_cast`, but I think it's a defensible practice here.

I would have named the function without the `_if_selected` and simply noted in the documentation that the function does nothing if no particle postprocessor has been selected